<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManagerEditorClass</class>
 <widget class="QMainWindow" name="TaskManagerEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManagerEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="testBoton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionXML"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionXML_2"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout_me">
    <property name="title">
     <string>About me</string>
    </property>
    <addaction name="actionGithub"/>
    <addaction name="actionLinkedin"/>
    <addaction name="separator"/>
    <addaction name="actionTask_Manager_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout_me"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionLinkedin">
   <property name="text">
    <string>Linkedin</string>
   </property>
  </action>
  <action name="actionTask_Manager_Editor">
   <property name="text">
    <string>Task Manager Editor</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionXML_2">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TaskManagerEditor.qrc"/>
 </resources>
 <connections/>
</ui>
